src/main/java/frc/team4909/robot/subsystems/elevatorarm/ElevatorArmSubsystem.java

```diff
@@ -60,6 +67,8 @@ public void reset(){
         elevatorArmSRX.setSelectedSensorPosition(0);
     }
     
+
+
     @Override
     public void periodic() {
         SmartDashboard.putNumber("Wrist position", elevatorArmSRX.getSelectedSensorPosition());
```

Comment:
There's no code in the periodic function other than displaying a value. Don't you need to check `getPosition() == 0` here, to stop the arm from moving? I don't see how `elevatorArmSetSpeed` gets called to do that check...???